src/utils/TestSuite.js

```diff
@@ -18,6 +18,7 @@ export default class TestSuite {
   start () {
     return new Promise((resolve, reject) => {
       return this.runNextTest().then(() => {
+        this.results = [];
```

Comment:
I think cleaning up this reference was the issue

Reply:
I'm not sure I understand. Why would you empty out the results?

Reply:
Well doesn't `this.results` instance get shared between the AdvancedCameraTest, AudioBandwidthTest, and the CameraResolutionTest, shouldn't the instance be cleaned up before each type of test. Once I cleared it, the test failures went away

Reply:
The results are populated by the test, and you're emptying the results as soon as the test finishes.